### DISCUSSION TITLE
Edge mover on Windows and/or Dialogs

### DISCUSSION THREAD

**Discussion by paule32** [magiblot/tvision#140] [Q&A] [answered]
Hello,
how can I add a "mover" on the blue Window in the Background, to change the Size of Window ?
In the Foreground, you can see the opener Window, and this Window have a "edge" mover, too.
(on the bottom right side).

![grafik](https://github.com/magiblot/tvision/assets/3682603/68dabfff-9c8a-46f4-9b96-78c19d41c07c)

**Comment by magiblot** [comment, marked as answer]
Hi Jens!

The sizing handles in Turbo Vision's original views are implemented directly in the `TFrame` source code:

https://github.com/magiblot/tvision/blob/0917f04206fcfa1973dc49547af74e7399241899/source/tvision/tframe.cpp#L117-L122

https://github.com/magiblot/tvision/blob/0917f04206fcfa1973dc49547af74e7399241899/source/tvision/tframe.cpp#L187-L194

So, if you are not seeing them, you should check:

* Are you using the `TFrame` class or a custom one without the code related to the sizing handles?
* If you are using `TFrame`, do your windows have the `wfGrow` flag set in the `TWindow::flags` field?

Cheers.

**Comment by paule32** [comment]
Thank You.
The Window Flag **wfGrow** was my Friend.
